### ISSUE TITLE
emb-w-04 - Data Table

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#5]
emb-w-04	Data Table	Yes	1-Critical	

Tables/grids cannot be navigated or sorted using keyboard/screen reader. 

Ensure that tables/grids are navigable and sortable using keyboard. Users should be able to traverse the tables in 2 dimensions and get header information when traversing between columns.

**Comment by bertramlyons** [comment]
Fixed as of version 1.3.2
